tools.py

```diff
@@ -0,0 +1,29 @@
+import getpass
+import requests
+import urllib
+import urlparse
+
+
+requests.packages.urllib3.disable_warnings()
+
+def vsphere_url(vm, host, args):
+    # Generates console URL for vSphere versions prior 6.0
```

Comment:
They are the orig dev's, so leaving as is. 
Since this is a fork of someone else's repo, our standards don't need to be met at the same level.